src/main/resources/odf1.3/OpenDocument-v1.3-part3-schema_odt/content.xml

```diff
@@ -14458,9 +14458,9 @@
-   <text:p text:style-name="Note">The number:max-denominator-value attribute specifies the maximum denominator permitted to be chosen if its &lt;number:fraction&gt; element does not have a number:denominator-value attribute. The number:max-denominator-value attribute is ignored in the presence of a number:denominator-value <text:reference-ref text:reference-format="chapter" text:ref-name="attribute-number:denominator-value">19.345</text:reference-ref><text:s/>attribute. The absence of the number:max-denominator-value attribute indicates that no maximum denominator is specified. </text:p>
-   <text:p text:style-name="Note"><text:line-break/><text:span text:style-name="Note_20_Label">Note:</text:span> The <text:span text:style-name="Attribute">number:max-denominator-value</text:span> attribute is used with number:grouping 19.348, <text:span text:style-name="Attribute">number:min-denominator-digits</text:span> 19.350, <text:span text:style-name="Attribute">number:min-integer-digits</text:span> 19.352 and <text:span text:style-name="Attribute">number:min-numerator-digits</text:span> 19.353 to define the display of a floating-point number as a fraction.</text:p>
-   <text:p text:style-name="Note"><text:line-break/><text:span text:style-name="Note_20_Label">Note:</text:span> A <text:span text:style-name="Attribute">number:max-denominator-value</text:span> attribute value of <text:span text:style-name="Attribute_20_Value">99</text:span> is equivalent to permitting any 1- or 2-digit denominator. A <text:span text:style-name="Attribute">number:max-denominator-value</text:span> attribute value of <text:span text:style-name="Attribute_20_Value">999</text:span> is equivalent to permitting any 1, 2, or 3 digit denominator. </text:p>
+   <text:p text:style-name="Text_20_body">The number:max-denominator-value attribute specifies the maximum denominator permitted to be chosen if its &lt;number:fraction&gt; element does not have a number:denominator-value attribute. The number:max-denominator-value attribute is ignored in the presence of a number:denominator-value <text:reference-ref text:reference-format="chapter" text:ref-name="attribute-number:denominator-value">19.345</text:reference-ref><text:s/>attribute. The absence of the number:max-denominator-value attribute indicates that no maximum denominator is specified. </text:p>
+   <text:p text:style-name="Text_20_body"><text:span text:style-name="Note_20_Label">Note:</text:span> The <text:span text:style-name="Attribute">number:max-denominator-value</text:span> attribute is used with number:grouping 19.348, <text:span text:style-name="Attribute">number:min-denominator-digits</text:span> 19.350, <text:span text:style-name="Attribute">number:min-integer-digits</text:span> 19.352 and <text:span text:style-name="Attribute">number:min-numerator-digits</text:span> 19.353 to define the display of a floating-point number as a fraction.</text:p>
```

Comment:
this one should still have style Note, the problem was the extra text:line-break